### PR TITLE
Finalize team section

### PR DESCRIPTION
**Pull request by galacemiguel**
**Description**
- Finalizes the core team section (closes https://github.com/reactph/reactjsph-website/issues/61)

**Summary of Changes**
- Updates member list
- Adjusts section layout for narrow displays
- Adds animation orbits background
- Changes team member hover animation

**How to Test**
- Verify that the changes satisfy https://github.com/reactph/reactjsph-website/issues/61

**Screenshots**
Kapture 2020-03-26 at 21 36 19

### CODE REVIEW

**Comment by galacemiguel** [comment]
> Can we remove the shaking orbit image in the background? In my opinion, it is kinda distracting?

@jromest Fair point. 🤔 I like it though because it makes this section feel more dynamic—it felt a bit dull to me before. How about the others?

@marjmartinezz @itsdyeyson

File: src/components/teamMember.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  member: { photo, name, role, website },
+  onMouseEnter: handleMouseEnter,
+  ...props
+}) => (
+  <Box
+    as="a"
+    href={website}
+    target="_blank"
+    rel="noopener noreferrer"
+    onMouseEnter={handleMouseEnter}
```
Comment (jromest) [inline src/components/teamMember.js]: I think`onMouseEnter` will not work on mobile. Add touch events or remove any hover action on mobile.

Comment (galacemiguel) [reply src/components/teamMember.js]: True. 🤔 I'd like to keep it though so at least if the user taps on someone's name, the effect will still show up.

**Comment by galacemiguel** [comment]
> - Gian's image is not square (1:1)
> - The background of the contact section is overlapping on mobile

Thanks for pointing these out, @jromest! 🙏 They should be fixed now.

**Pull request merged**